rocketmq-connect/pom.xml

```diff
@@ -40,7 +40,7 @@
         <assertj.version>2.6.0</assertj.version>
         <mockito.version>2.6.3</mockito.version>
         <httpclient.version>4.5.5</httpclient.version>
-        <openmessaging.connector.version>0.1.0-beta</openmessaging.connector.version>
```

Comment:
the openmessaging connect 0.1.0 will be released soon.

Reply:
@duhenglucky done